Sources/KIF/Additions/UIAccessibilityCustomAction+KIFAdditions.m

```diff
@@ -0,0 +1,46 @@
+//
+//  UIAccessibilityCustomAction+KIFAdditions.m
+//  KIF Tests
+//
+//  Created by Alex Odawa on 09/07/2024.
+//
+
+#import <UIKit/UIKit.h>
+#import "UIAccessibilityCustomAction+KIFAdditions.h"
+
+@implementation UIAccessibilityCustomAction (KIFAdditions)
+
+- (BOOL)activate;
+{
+    if (@available(iOS 13.0, *)) {
+        if (self.actionHandler && self.actionHandler(self)) {
```

Comment:
Assuming it's intentional we don't return the result of `actionHandler` here?

Reply:
the docs say that if theres an action handler it takes precedence, but nothing prevents a UIAccessibilityCustomAction from having both an actionHandler and a target/selector pair. 

This allows us to give the target/selector (if it exists) an opportunity to succeed (return true) if the actionHandler doesn't..